CrowServe/tests/CrowServeTest.cpp

```diff
@@ -4,6 +4,17 @@
 #include "Cbor.h"
 #include <condition_variable>
 
+template <typename T>
+struct ReceiveWrapper {
```

Comment:
This seems odd, but since it's test so feel free to leave it. Probably should have been defined inline or defined as class with methods and no outside mutation.

Reply:
Yeah it's a bit hacky but just for testing I needed a way to 'script' actions and wait for responses. I'm sure there's a better way but it was more convenient than creating a mutex & cv separate for each message type I needed to test.